http/src/main/resources/init-resources/startup.sh

```diff
@@ -64,6 +64,17 @@ export WELDER_MEM_LIMIT=$(welderMemLimit)
 export MEM_LIMIT=$(memLimit)
 export USE_GCE_STARTUP_SCRIPT=$(useGceStartupScript)
 
+# Overwrite old cert on restart
+SERVER_CRT=$(proxyServerCrt)
```

Comment:
Do these match keys in `RuntimeTemplateValues`, and are they replaced at runtime resume time? (I recall some templated values are optional and are only populated at creation time.)

Reply:
oh didn't realize some values are optional...I can check..maybe that's why automation test failed